internal/ui/components/sanitize_test.go

```
@@ -38,6 +38,71 @@ func TestSanitizeStripsControlCharacters(t *testing.T) {
+			contain: []string{"before", "after"},
+		},
+		{
+			name:   "bel backspace del removed",
+			in:     "a\x07b\x08c\x7fd",
+			absent: []rune{0x07, 0x08, 0x7f},
+			// The letters between the control bytes survive.
+			contain: []string{"a", "b", "c", "d"},
+		},
+		{
+			name:    "c1 csi byte removed",
+			in:      "x31my",
```

golangci-lint (2.12.2)

[error] ST1018: string literal contains the Unicode control character U+009B, consider using the '\u009b' escape sequence instead

(staticcheck)